apps/backend/src/routes/login/+page.svelte

```diff
@@ -0,0 +1,161 @@
+    }),
+
+    onSubmit: async (data) => {
+      console.log(data);
+      try {
+        await signInEmailPassword(data.email, data.password, (error) => (apiError = error));
+      } catch (error) {
+        console.error(error);
+      }
+    },
+  });
+</script>
+
+<!-- Armadillo Logo-->
+<div class="flex items-center justify-center p-10">
+  <img src="armadillo.png" alt="" />
+</div>
+
+<div class="flex items-center justify-center">
+  <!-- Login Form - Card -->
+  <div class="w-full max-w-lg p-10 border border-secondary rounded-md shadow-lg">
+    <!-- Login Form - Header -->
+    <p class="text-3xl font-semibold text-center mb-5">Welcome Back</p>
+
+    <!-- Login Form - Form Content -->
+    <form on:submit|preventDefault={handleSubmit}>
+      <!-- General Error Message -->
+      {#if apiError && (apiError.code === 'auth/user-not-found' || apiError.code === 'auth/wrong-password')}
+        <div class="p-4 flex flex-row bg-red-50 text-red-800 rounded-lg" role="alert">
+          <img src="errorlogo.svg" alt="" />
+          <div class="ml-2 text-sm font-medium">Invalid email or password, please try again</div>
+        </div>
+      {/if}
+
+      <!-- Form Content - Email Input -->
+      <div>
+        <label for="email" class="form-control w-full max-w-md">
+          <div class="label">
+            <span class="label-text">Email</span>
+          </div>
+          <input
+            id="email"
+            type="text"
+            name="email"
+            placeholder="Name@example.com"
+            on:change={handleChange}
+            bind:value={$form.email}
+            class="
+							input
+							input-bordered
+							w-full
+							max-w-md
+              focus:ring-secondary
+							focus:border-secondary
+							duration-300
+							focus:outline-none
+						" />
+        </label>
+        <div>
+          <span class="text-red-600">{$errors.email}</span>
+        </div>
+      </div>
+
+      <!-- Form Content - Password Input -->
+      <div>
+        <label for="password" class="form-control w-full max-w-md">
+          <div class="label">
+            <span class="label-text">Password</span>
+          </div>
+          <input
+            id="password"
+            type="password"
+            name="password"
+            placeholder="••••••••"
+            on:change={handleChange}
+            bind:value={$form.password}
+            class="
+							input
+							input-bordered
+							w-full
+							max-w-md
+              focus:ring-secondary
+							focus:border-secondary
+							duration-300
+							focus:outline-none
+						" />
+        </label>
+        <div>
+          <span class="text-red-600">{$errors.password}</span>
+        </div>
+      </div>
+
+      <!-- Form Content - Submit/Continue Button -->
+      <button
+        type="submit"
+        class="btn
+					btn-secondary
+					w-full
+					max-w-2xl
+					mt-8
+					mb-2
+					hover:ring-2
+					hover:ring-info"
+        >Continue
```

Comment:
Keep your `>` on the same line, and `Continue` on another line.

E.g.,
```
<button
  type="submit"
  ...>
    Continue
</button>
```

Reply:
Changed accordingly